### ISSUE TITLE
Summary documentation shall use '/// <summary>' instead of '// Summary:'

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#1347]
The API documentation for summaries should use `/// <summary>` instead of `// Summary:` because `/// <summary>` creates XML comments that the .NET compiler understands. These comments show up in IntelliSense, help generate external documentation, and follow a structured format.

`// Summary:` will not be used by tools or IDEs to provide helpful info.

**Comment by RalfKoban** [comment]
So following comment should trigger a warning.

```c#
        // Summary:
        //     Gets the error message for the property with the given name.
        //
        // Parameters:
        //   columnName:
        //     The name of the property whose error message to get.
        //
        // Returns:
        //     The error message for the property. The default is an empty string ("").
```

It should be fixed into:
```c#
        /// <summary>
        /// Gets the error message for the property with the given name.
        /// </summary>
        /// <param name="columnName">
        /// The name of the property whose error message to get.
        /// </param>
        /// <returns>
        /// The error message for the property. The default is an empty string ("").
        /// </returns>
```
